lib/zclient.h

```diff
@@ -346,6 +346,7 @@ enum zapi_route_notify_owner {
 	ZAPI_ROUTE_FAIL_INSTALL,
 	ZAPI_ROUTE_BETTER_ADMIN_WON,
 	ZAPI_ROUTE_INSTALLED,
+	ZAPI_ROUTE_REMOVED,
```

Comment:
What about notifying when a route failed to be removed (e.g. `ZAPI_ROUTE_FAIL_REMOVE`)? It probably wouldn't be very useful but maybe we can add it for completeness sake.

Reply:
I just happened to be running around like a chicken this afternoon when I wrote this and was focusing on what I really really needed.  I've added the new case in now